src/frontend/apps/hub/src/features/chat/components/ChatBubble.scss

```
@@ -120,12 +122,15 @@
   }
 
   &--received &__footer {
-    max-width: calc(400px + 24px + var(--c--globals--spacings--xs, 8px));
+    max-width: calc(
+      var(--hub-chat-bubble-max-width) + 24px +
```

Stylelint (17.14.0)

[error] Unexpected newline after "+" (scss/operator-no-newline-after)

(scss/operator-no-newline-after)